WordPress/gutenberg
# removing the extra padding in background settings in site tittle block

## What?

This PR will solve the issue of extra padding added when the background color is selected in the site title block.

## Why?

In FSE editor when we give the background color to the site title block then extra padding is added. And it is not correct. On the front side, this issue is not happening only on the editor side this is happening. I found the extra CSS due to which the issue is happening. 

## How?

So right now I have made PR of removal of that CSS but any suggestion for improvement in CSS will be a great help.

## Testing Instructions

1. Open FSE.
2. Select the site title block in the nav menu.
3. Add the background color and you can see some padding will be added with background-color

## Screenshots or screencast
